atomium/includes/common.inc

```diff
@@ -646,8 +646,6 @@ function atomium_get_base_hook($hook) {
  * Discover templates based on directory name.
  *
  * It is important that directories have the name of the hook for the moment.
- *
- * @deprecated Use atomium_discover_templates()
  */
 function atomium_find_templates() {
```

Comment:
If `atomium_find_templates()` is not used elsewhere, please remove it completely then.